subjects/PE-x86/VCExeSample/VCExeSample.c

```diff
@@ -10,9 +10,9 @@ int32 main(int32 argc, char * * argv)
 	return 0x00;
 }
 
-void test1(char * arg1, int32 arg2, char * arg3, real32 arg4)
+void test1(char * arg1, int32 arg2, char * arg3, Eq_10 arg4)
 {
-	printf("%s %d %s %f");
+	printf("%s %d %s %f", arg1, arg2, arg3, arg4);
 	return;
```

Comment:
Few months ago you have fixed x86 rewriter so that it returns cast for `fstp` if operands sizes are different (see #228). I think the same fix should be applied to `fld` and `Eq_10` will go away